packages/hub-nodejs/src/scripts/object_test.ts

```diff
@@ -65,13 +77,48 @@ const NETWORK = FarcasterNetwork.DEVNET; // Network of the Hub
   ed25519Signer);
 
   console.log('ObjectAdd message', objectAdd);
-  const tag = await client.submitMessage(objectAdd._unsafeUnwrap());
-  console.log(tag._unsafeUnwrap().data?.objectAddBody);
+  const object = await client.submitMessage(objectAdd._unsafeUnwrap());
+
+  const tagAdd = await makeTagAdd({
+    name: 'newtagg',
+    content: 'the best',
+    target: {
+      castKey: {
+        network: 3,
+        hash: objectAdd._unsafeUnwrap().hash,
+        fid: FID,
+      },
+    }
```

Comment:
will need to update this test file to follow the new `ObjectRef` structure for tag target